controllers.js

```diff
@@ -43,7 +44,18 @@ exports.getArticleById = (req, res, next) => {
 exports.getCommentsByArticleId = (req, res, next) => {
   selectCommentsByArticleId(req)
     .then((comments) => {
-        res.send({ comments });
+      res.send({ comments });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.postCommentByArticleId = (req, res, next) => {
+  insertCommentByArticleId(req)
```

Comment:
I wouldn't recommend passing the req into the model, it means that your model now cares about how your controller is being invoked and what with meaning that if we were to change something about the path or the server itself we would have to shoe horn in changes to the model. I would probably have this function be invoked with the article_id and an object that represents the keys that are to be added to the database